packages/vis/src/composables/__tests__/useTimeline.spec.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @jest-environment jsdom
+ */
+ import { describe, expect, it } from 'vitest'
+
+import { useTimeline } from '../../'
```

Comment:
should we use full path here?

Reply:
So I like to test that we're resolving from the route index.ts of the project, so we know it will be defined in our final package output.

Reply:
Watch this space - more tests to come*

Reply:
Alright! thanks for the answer